jsapp/js/searches.es6

```diff
@@ -94,7 +93,22 @@ function SearchContext(opts={}) {
           }
         }
       };
+      var filterOutDeletedAssetFromCategorizedList = () => {
+        let list = this.state.defaultQueryCategorizedResultsLists;
```

Comment:
Do we have any rules on `const` vs `let`? There is an eslint rule about that: https://eslint.org/docs/rules/prefer-const

Reply:
We currently don't. I'd rather not change this now, let's note it for a future formatting/code cleanup update.